scripts/lib/CIME/SystemTests/system_tests_compare_two.py

```diff
@@ -53,7 +53,8 @@ def __init__(self,
                  run_two_suffix = 'test',
                  run_one_description = '',
                  run_two_description = '',
-                 multisubmit = False):
+                 multisubmit = False,
+                 expected=None):
```

Comment:
Can you please add some documentation of what this "expected" argument is for?

Actually, from a quick read of the code, I wonder if this is really needed now: It looks like this is just needed for working with locked files, and I'm wondering if that is no longer needed now that you're using the system_tests_compare_two infrastructure.